python/lsst/scarlet/lite/initialization.py

```diff
@@ -108,31 +108,38 @@ def init_monotonic_morph(
     morph:
         The initialized morphology.
     """
+    center: tuple[int, int] = tuple(center[i] - full_box.origin[i] for i in range(2))  # type: ignore
```

Comment:
No, it's worse than that (IMHO). `Box` can be an n-dimensional box, so the type of `Box.origin` is `[int, ...]`, but some of the methods called in `init_monotonic_morph` required `[int, int]` (in other words `center` has to be 2D). Mypy does not (for better or for worse) recognize that `[int, ...]` also satisfies `[int, int]`, because of course it could contain a tuple with more than 2 entries.

I could cast it here, but I'm fully frustrated with mypy at this point and I don't feel like fighting it anymore. Mypy was really great for finding bugs in new code, especially in a new package, but when things don't just work I've decided to ignore types rather than waste time making mypy happy when it doesn't add anything to the code.